amfora.go

```diff
@@ -68,10 +70,31 @@ func main() {
 	display.NewTab()
 	if len(os.Args[1:]) > 0 {
 		display.URL(os.Args[1])
+	} else if !isStdinEmpty() {
+		renderFromStdin()
 	}
 
 	// Start
 	if err = display.App.Run(); err != nil {
 		panic(err)
 	}
 }
+
+func isStdinEmpty() bool {
+	stat, _ := os.Stdin.Stat()
+	return (stat.Mode() & os.ModeCharDevice) != 0
+}
+
+func renderFromStdin() {
+	stdinTextBuilder := new(strings.Builder)
+	_, err := io.Copy(stdinTextBuilder, os.Stdin)
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "error reading from standard input: %v\n", err)
+		os.Exit(1)
+	}
+
+	stdinText := stdinTextBuilder.String()
+	if len(strings.TrimSpace(stdinText)) > 0 {
```

Comment:
I don't think this check is necessary. Nothing wrong with displaying some spaces or just nothing at all. Would be useful if part of some longer pipeline, where returning nothing is possible but not certain. Much better to get an empty page than be confused by the new tab page.

Other than this I think we're good to go!